services/orchestrator/src/main/java/org/eclipse/osc/orchestrator/OrchestratorPlugin.java

```diff
@@ -0,0 +1,39 @@
+package org.eclipse.osc.orchestrator;
+
+import org.eclipse.osc.services.ocl.loader.Ocl;
+
+/**
+ * This interface describes orchestrator plugin in charge of interacting with backend fundamental APIs.
```

Comment:
For the "existing" API (like compute/network/storage), I agree. However, I think it won't be enough: how can we do billing integration, admin console integration, monitoring/logging integration for instance ?
Does it mean that OCL would be a mix of IaC and additional metadata ? I see more IaC translation in the orchestrator plugin but both are possible.
Actually, I was thinking why not defining all with `properties` and then translate in the Orchestrator plugin.

Do you have kind of example snippet in mind ?

Reply:
actually, I still not quite sure about how we can do billing/console integration for now, so just raise the idea of IaC translation. it would be better to do some further discussion about this after the demo show.
If biling/console integration process are different with cloud service integration, Iac translation could make sense

Reply:
Yes, it makes sense.And I agree: let's move forward with the "know how it works" things ;)